src/jsx.d.ts

```diff
@@ -750,6 +750,7 @@ export namespace JSXInternal {
 		ul: HTMLAttributes;
 		"var": HTMLAttributes;
 		video: HTMLAttributes;
+		volume: HTMLAttributes;
```

Comment:
This should be in HTMLAttributes instead.
```typescript
interface HTMLAttributes {
	...
	volume?: string | number
}
```

Reply:
Right you are. New commit has this change, removed from Intrinsic Elements.